src/test/Home.test.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
```

Comment:
- [ ] You have failing tests from `Home.test.js` and `Navbar.test.js`. Please make sure all your tests are passing.
![image](https://github.com/MargaretTechworld/React-Redux-Solo-Capstone/assets/45460192/1d52245b-1ed5-47e9-943e-9ce89baef051)

Reply:
Hi, my tests are passing on my side
![image](https://github.com/MargaretTechworld/React-Redux-Solo-Capstone/assets/60655304/a0a74262-a70c-4935-bde0-13521de2a8f1)